android/app/src/main/java/com/bounswe2020group3/paperlayer/project/ProjectDetailFragment.kt

```diff
@@ -192,11 +192,12 @@ class ProjectDetailFragment : Fragment(),ProjectDetailContract.View {
             this.fragmentView.textViewEvents.text=project.event.title
         }
 
-        //Adding Members
-        for (member in project.members!!){
-                addMemberCard(member.username)
-                submitMemberCardList()
-        }
+
+//        //Adding Members
```

Comment:
Normally, you shouldn't just comment out the code but I'm fixing this bug in another branch, therefore it is okay.

Reply:
yes, I know but we talked with @rambo250 about it and i didnt want to delete his code completely without noticing.